test/src/modules/serializer/component/__tests__/component.spec.js

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import { createElement } from 'lwc';
+import Component from '../component';
+
+it('resolves component without HTML', () => {
```

Comment:
The test component does have a template.
```suggestion
it('serializes component with HTML', () => {
```